hctl

```diff
@@ -133,6 +133,8 @@ case $cmd in
         PATH="$HARE_BASE_DIR/libexec:$PATH"
         PATH="$HARE_BASE_DIR/bin:$PATH"
         export PATH
+        export LANG="en_US.UTF-8"
```

Comment:
@mssawant Do we need to check if locale is available before using it? We are doing that in hax.py above.

Reply:
@SwapnilGaonkar7, added a check, kindly review the updated changes.